tensorflow/python/ops/image_ops_impl.py

```diff
@@ -345,17 +345,6 @@ def random_flip_up_down(image, seed=None):
     ... ])
     >>> tf.image.random_flip_up_down(images, 4).numpy().tolist()
```

Comment:
I just noticed that there are actually usage examples for this and the next function. I couldn't decide on which one to keep so could you please choose one so I could delete the other one?

Reply:
No need to add examples if there are some already

Reply:
Ok, removed the extra examples with commit https://github.com/tensorflow/tensorflow/pull/35388/commits/0a89b29136c8233d521bf26450c8492fa72576a1 . Can you now check my whole PR once again?